src/api-service/__app__/onefuzzlib/events.py

```diff
@@ -33,9 +50,54 @@ def get_events() -> Optional[str]:
         return None
 
 
+def log_event(event: Event, event_type: EventType) -> Event:
```

Comment:
```suggestion
def log_event(event: Event, event_type: EventType) -> None:
```

Reply:
Rather than make `log_event` return an Event, which feels weird, I suggest a `filter_event` function which does the underlying work.

Reply:
I only include the return function for the unit tests. I'm not sure I see the use in adding a third function?